cugraph-analytics/README.md

```diff
@@ -0,0 +1,36 @@
+# Run Graph Analytics on Large Scale Graphs with NVIDIA and Memgraph 
+This tutorial will show you how to use  **PageRank** graph analysis and **Louvain** community detection on a **Facebook dataset** containing 1.3 million relationships. 
+Upon completing it, you will know how to run analytics algorithms on your dataset using **Python**. Then, you can run any of the following algorithms:
+* Balanced Cut (clustering)
+* Spectral Clustering (clustering)
+* HITS (hubs vs. authorities analytics)
+* Leiden (community detection)
+* Katz Centrality
+* Betweenness Centrality
+
+All of the algorithms above are powered by **[Nvidia cuGraph](https://www.nvidia.com/en-us/)** and they will execute on **GPU**.
+
+## Prerequisites 
+
+To follow the tutorial, please install:
+- [Docker](https://docs.docker.com/get-docker/) - needed to run `mage-cugraph` image
```

Comment:
`mage-cugraph` is not the final name of the Docker image, update each occurrence